logstash.go

```diff
@@ -56,8 +57,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 		}
 		_, err = a.conn.Write(js)
 		if err != nil {
-			log.Println("logstash:", err)
-			continue
+			log.Println("fatal logstash:", err)
```

Comment:
No particular reason. Does `Fatalln` exit or do I need to explicitely do `os.Exit(1)`?